dask_expr/io/io.py

```diff
@@ -132,6 +133,12 @@ def _task(self, index: int):
         bucket = self._fusion_buckets[index]
         return (methods.concat, [expr._filtered_task(i) for i in bucket])
 
+    def _simplify_down(self):
```

Comment:
I would rather overwrite the tune_up method in ReadParquetPyarrowFS, but that's not blocking